dgraph/cmd/root.go

```diff
@@ -74,6 +74,8 @@ func init() {
 		"Use 0.0.0.0 instead of localhost to bind to all addresses on local machine.")
 	RootCmd.PersistentFlags().Bool("expose_trace", false,
 		"Allow trace endpoint to be accessible from remote")
+	RootCmd.PersistentFlags().Bool("enterprise_features", false,
+		"Enable enterprise features")
```

Comment:
Enable Dgraph enterprise features. If you set this to true, you agree to the Dgraph community license.

Reply:
s/Dgraph community license/Dgraph Community License.